### ISSUE TITLE
Player stuck after quality change and seek in rapid succession

### ISSUE THREAD

**Issue by Vox94** [videojs/video.js#7107]
## Description
Briefly describe the issue.
Include a [reduced test case](https://css-tricks.com/reduced-test-cases/), we have a [starter template](https://codepen.io/gkatsev/pen/GwZegv?editors=1000#0) on JSBin you can use.

[Reduced test case](https://codepen.io/Vox94/pen/MWbObXM?editors=1001)
**!NB -** I ended up copy-pasting the **ENTIRE** contrib quality levels in the file, as for some reason using a script src tag did not work for me. However, the contrib quality levels that is pasted in there is **UNMODIFIED** 

When seeking backwards, if the video hasn't buffered enough, we ender an infinite loop.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Go to some moment on video (e.g. 70%)
2. Switch to some quality (e.g. 1080p) (wait for first frame to load)
3. Instantly switch to other quality (e.g. 240p)
4. Instantly move progress bar to some earlier point in the video (e.g. 30%)

Notes: The video resumes if you go back to the video at the moment of the quality switch (in point 1), or any other time after that point (in the case of point 1 - 70% or after)

### Error output
No errors are given whatsoever

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
Tested on some versions > [7.10](https://github.com/videojs/video.js/blob/main/CHANGELOG.md#7100-2020-10-14) (including latest, [7.11.5](https://github.com/videojs/video.js/blob/main/CHANGELOG.md#7115-2021-02-04))
#### browsers
what browser are affected? 
Google chrome v88.0.4324.182
Mozilla Firefox v85.0.2
#### OSes
what platforms (operating systems and devices) are affected?
Tested in Win10 - version 10.0.19041 Build 19041
(I have not checked in on phones, as it would be marginally difficult to fulfill the same steps there within such a short time period)
### plugins
are any videojs plugins being used on the page? If so, please list them below.
`videojs-contrib-quality-levels @version 2.0.9
videojs-hls-quality-selector @version 1.1.4

**Comment by AlekseyChepurko** [comment]
hey guys. any news here?

**Comment by gkatsev** [comment]
I'm unable to reproduce. It's currently loading 7.12.3.

**Comment by Vox94** [comment]
Indeed. I do seem to no longer be able to reproduce it with the steps I provided in 7.12.3. However, switching to a different moment in the video, changing the resolution, switching again and changing again in rapid succession (basically repeated that for 5-6 times, as fast as I could) does indeed seem to eventually reproduce it.
I'm attaching a visual with the case.
![OMf5dYZ7q9](https://user-images.githubusercontent.com/29002510/123425077-5587b800-d5ca-11eb-8044-51cc20cc1cba.gif)
https://imgur.com/a/pbNpOY0
(the second one is a link, as it is a bit too large to upload to github)

**Comment by Vox94** [comment]
This should still be active imo, just waiting for a reply.

**Comment by Vox94** [comment]
Still waiting

**Comment by privaloops** [comment]
Same here, please don't close this ticket

**Comment by privaloops** [comment]
same here : https://github.com/videojs/http-streaming/issues/1273

**Comment by privaloops** [comment]
Hello, any news ?

**Comment by therealabdi2** [comment]
@privaloops were you able to solve this?
